addons/utm/models/utm_mixin.py

```diff
@@ -38,7 +38,8 @@ def default_get(self, fields):
                     value = request.httprequest.cookies.get(cookie_name)
                 # if we receive a string for a many2one, we search/create the id
                 if field.type == 'many2one' and isinstance(value, str) and value:
-                    record = self._find_or_create_record(field.comodel_name, value)
+                    lowercase = url_param != 'utm_campaign'
```

Comment:
Might make sense to set "lower" as a data point in `tracking_fields` instead, otherwise it looks a bit arbitrary (well it is, but it feels weird to do arbitrary stuff in logic parts of the code)